packages/terra-search-field/tests/nightwatch/SearchFieldDefaultValue.jsx

```diff
@@ -1,5 +1,10 @@
 import React from 'react';
-
+import Base from 'terra-base';
 import SearchField from '../../lib/SearchField';
 
-export default () => <SearchField id="searchfield" defaultValue="Default" />;
+const locale = document.getElementsByTagName('html')[0].getAttribute('lang');
```

Comment:
I notice we use this to get our locale a lot. What are people's opinions of attaching a static method to base that gets the value? It would look like this `Base.getCurrentLocale();`

Reply:
I know we have a utilities package lined up, so it could go there as well.

Reply:
Actually, given the new site implementation, the need to add the base component for test example should no longer be needed